src/main/java/net/imagini/zkcli/ZkCli.java

```diff
@@ -51,6 +51,7 @@ private void doCliActions() {
                                                       connectionMethodInvalidException);
                 }
             }
+            // TODO: don't assume a cluster is always specified
```

Comment:
This actually isn't assumed. At the top of the doCliActions method I try and build a client from the connection string or friendly name. If neither are provided it prints "Please provide a valid connection string or cluster alias". e.g.
 
    java -jar target/JZookeeperEdit.jar -l /
    Please provide a valid connection string or cluster alias